internals/plan/plan.go

```diff
@@ -536,6 +536,7 @@ func (e *FormatError) Error() string {
 // layers overriding earlier ones.
 func CombineLayers(layers ...*Layer) (*Layer, error) {
 	var errorMessages []string // To collect multiple errors
+	addError := func(message string) { errorMessages = append(errorMessages, message) }
```

Comment:
since addError is always called with fmt.Sprintf, let's pull the fmt.Sprintf call inside addError, and make it addErrorf

Reply:
@barrettj12 
Is there anything else ??